New package: arc-themes-20151214

pbui opened the pull request.

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,21 @@
+pkgname="arc-themes"
```
First line should be a comment `# Template file for 'arc-themes'`. pkgname should not have quotes around it. Neither should version.

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,21 @@
+build_style="gnu-configure"
+hostmakedepends="pkg-config autoconf automake"
+makedepends="gtk+-devel gtk+3-devel"
+depends="gtk+ gtk2-engines gtk-engine-murrine gnome-themes-standard"
```
Why depend on a specific gtk? The theme works for either.

pbui commented:
@Vaelatern I fixed the quotes and added the comment.  I'm not sure what you mean by depending on a specific version of gtk+, but I removed gtk2-engines.  From the upstream website, the theme requires gtk+, gtk-engine-murrine, and gnome-themes-standard.  To build and auto-detect the correct gtk+ versions, I believe `makedepends` requires both `gtk+-devel` and `gtk+3-devel`, but I can change that if necessary.

Vaelatern commented:
That part is fine, I just don't think you need to add them as runtime dependencies.

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,22 @@
+noarch=yes
+wrksrc="arc-theme-${version}"
+short_desc="A flat theme with transparent elements for GTK 2 and 3"
+build_style=fetch
```
Remove this line.

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,22 @@
+pkgname=arc-themes
+version=20151214
+revision=1
+noarch=yes
```
Remove this line.

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,22 @@
+build_style="gnu-configure"
+hostmakedepends="pkg-config autoconf automake"
+makedepends="gtk+-devel gtk+3-devel"
+depends="gtk+ gtk-engine-murrine gnome-themes-standard"
```
`depends="gtk-engine-murrine gnome-themes-standard"`

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,22 @@
+license="GPL-3"
+noarch=yes
+build_style="gnu-configure"
+hostmakedepends="pkg-config autoconf automake"
```
`hostmakedepends="pkg-config automake"`

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,22 @@
+# Template file for 'arc-themes'
+pkgname=arc-themes
+version=20151214
```
Latest release is `20160331` and needs a new checksum for it. Fixes for GTK 3.20.

Vaelatern replied:
To be fair, the release updated after you opened this PR.

pbui commented:
I've made the changes you've requested.  Sorry about the sloppiness.

Vaelatern commented on srcpkgs/arc-themes/template:
```diff
@@ -0,0 +1,20 @@
+pkgname=arc-themes
+version=20160331
+revision=1
+noarch=yes
```
Remove this line

Vaelatern replied:
Disregard.

Vaelatern commented:
#4002 was closed, so this shall be as well.

xtraeme commented:
I refuse to push any THEME available on inet just because it can be packaged.

This theme can be downloaded by the user and unpacked into its home directory.

There's no need to make an xbps package.

Thanks!
